cli/import/examples/example.sql

```diff
@@ -4,8 +4,8 @@ Sample DB for the import command. You can create an sqlite DB with the command:
 $ sqlite3 example.db < example.sql
 */
 CREATE TABLE files (
-		blob_id_a TEXT, repository_id_a TEXT, commit_hash_a TEXT, path_a TEXT, content_a TEXT,
-		blob_id_b TEXT, repository_id_b TEXT, commit_hash_b TEXT, path_b TEXT, content_b TEXT,
+		blob_id_a TEXT, repository_id_a TEXT, commit_hash_a TEXT, path_a TEXT, content_a TEXT, uast_a BLOB,
+		blob_id_b TEXT, repository_id_b TEXT, commit_hash_b TEXT, path_b TEXT, content_b TEXT, uast_b BLOB,
```

Comment:
Type blob is not supported in postgres

Reply:
Since this is an internal tool, and (internally) we're not using PostgreSQL, I'd not block this PR because of this problem with PostgreSQL.
If Blobs cannot be easily solved for PostgreSQL, I'd merge as it is, and create a new issue to solve it when facing [`v1`](https://github.com/src-d/code-annotation/labels/v1)

Reply:
@carlosms nice catch! 

@carlosms @smacker  Is there anything we could use instead that would allow to keep v compatibility?

Reply:
We could do something similar to this:
https://github.com/src-d/code-annotation/blob/master/server/dbutil/dbutil.go#L38

And use [`bytea` type for postgres](https://www.postgresql.org/docs/current/static/datatype-binary.html). We'd have to check if it works different or has any other side effect.

Reply:
If it requires extra work, I'd not block the merge as explained at https://github.com/src-d/code-annotation/pull/206#pullrequestreview-103163429

Reply:
Nice! Thank you for pointing it out @carlosms 

So, if that (and that's how it seems from example above) would require some custom code, to change the query at the runtime by replacing the type, right?

Reply:
That's right. For the `<INCREMENT_TYPE>` the substitution is done [in this method](https://github.com/src-d/code-annotation/blob/master/server/dbutil/dbutil.go#L142).

Reply:
done by:
- 836d8db Add PostgreSQL support for BLOB